specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```diff
@@ -3339,12 +3339,29 @@
           "type": "integer",
           "default": 60,
           "description": "Monitoring interval in seconds."
-        }
+        },
+        "endpoints": {
+          "type": "array",
+          "description": "List of connection monitor endpoints.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorEndpoint"
+          }
+        },
+        "testGroups": {
+          "type": "array",
+          "description": "List of connection monitor tests.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorTestGroup"
+          }
+        }, 
+        "testConfigurations": {
+          "type": "array",
+          "description": "List of connection monitor test configurations.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorTestConfiguration"
+          }
+        }      
```

Comment:
Please remove trailing space

Reply:
sorry, didn't get it. What exactly I should remove?

Reply:
Extra space after bracket
![image](https://user-images.githubusercontent.com/8080186/59248694-d4039e00-8c2b-11e9-84fd-8ab96eebe11a.png)